charts/alfresco-ai-transformer/templates/deployment-ai-transformer.yaml

```diff
@@ -31,17 +31,17 @@ spec:
               name: {{ template "alfresco-ai-transformer.fullname" . }}
           env:
             {{- $mqSecret := coalesce .Values.messageBroker.existingSecret.name (include "alfresco-ai-transformer.secret-ai-mq.name" .) }}
-            - name: BROKER_USERNAME
+            - name: ACTIVEMQ_USER
               valueFrom:
                 secretKeyRef:
                   name: {{ $mqSecret }}
                   key: {{ .Values.messageBroker.existingSecret.keys.username }}
-            - name: BROKER_PASSWORD
+            - name: ACTIVEMQ_PASSWORD
               valueFrom:
                 secretKeyRef:
                   name: {{ $mqSecret }}
                   key: {{ .Values.messageBroker.existingSecret.keys.password }}
-            - name: BROKER_URL
+            - name: ACTIVEMQ_URL
```

Comment:
![](https://i.guim.co.uk/img/media/74e5a132a35ede8e1c2b8d3af3313518e2b84e70/0_0_1800_1080/master/1800.jpg?width=465&dpr=1&s=none)

Reply:
I overlooked it:
https://github.com/Alfresco/alfresco-ai-renditions/blob/master/ai-renditions/docker-compose/docker-compose.yml

Reply:
yeah, but the application should be complaining about a missing mq config given it can do anything useful without

Reply:
yeah but it doesn't. Regardless on whether the mq connection is set up or not... it seems.
As explained in the JIRa comment I still need to review the logs and config with @mstrankowski (hence the `-alpha.6`)